### ISSUE TITLE
Create new release

### ISSUE THREAD

**Issue by TimoGlastra** [Sphereon-Opensource/pex-openapi#37]
Would it be possible to create a new release including #36?

**Comment by nklomp** [comment]
Yeah, we have another ticket about field optionality, that I want to include at least in the models: https://github.com/Sphereon-Opensource/PEX/issues/150
